### ISSUE TITLE
Fix accessibility tests (pa11y)

### ISSUE THREAD

**Issue by beechnut** [18F/NRM-Grants-Agreements#334]
Pa11y-ci is running accessibility tests in CircleCI to ensure Sec 508/WCAG compliance.

At the moment, we're ignoring the exit code of the task and sending `exit 0` to allow the build to succeed while accessibility tests are failing. Ultimately, failed accessibility tests should result in a failed build.

**We'll know we're done when...**
- [ ] Accessibility tests for the existing pages are passing.
- [ ] We are sending the exit code of pa11y to CircleCI.

**Tasks**
- [ ] Add a comprehensive set of URLs to pa11y (the index, show, and edit pages for all existing resources)
- [ ] Run pa11y to determine accessibility problems.
- [ ] Fix the accessibility problems so pa11y returns no failures.
- [ ] Remove the exit code negation (the `!` in front of `npm run test:a11y`)
